app/controllers/state_file/questions/waiting_to_load_data_controller.rb

```diff
@@ -7,6 +7,7 @@ class WaitingToLoadDataController < QuestionsController
       def edit
         raise ActionController::RoutingError, 'Not Found' unless params[:authorizationCode]
         return redirect_to next_path if current_intake.raw_direct_file_data.present?
+        return redirect_to IntakeLoginsController.to_path_helper(action: :new, us_state: "us") unless current_intake.consented_to_terms_and_conditions
```

Comment:
There is no protection on the return from IRS so we check that the intake consented!